Corrected typos and added punctuation marks to sentences missing them. (#9422)
wagtail/wagtail: docs/getting_started/integrating_into_django.md

# Integrating Wagtail into a Django project

> Wagtail provides the `wagtail start` command, and project template, to get you started with a new Wagtail project as quickly as possible. However, It is easy to integrate Wagtail into an existing Django project as well.

Reviewer: I’m not finding this reads any better to be honest.

```suggestion
Wagtail provides the `wagtail start` command and project template to get you started with a new Wagtail project as quickly as possible, but it's easy to integrate Wagtail into an existing Django project too.
```
